packages/cbioportal-utils/src/format/FormatUtils.ts

```diff
@@ -34,9 +34,11 @@ export function formatNumberValueInSignificantDigits(
         return null;
     } else if (significantDigits) {
         return Number(
-            value.toLocaleString(undefined, {
-                maximumSignificantDigits: significantDigits,
-            })
+            value
+                .toLocaleString('en-US', {
```

Comment:
What does `en-US` do?

Reply:
Not sure if this is critical but this is basically to make sure that we are using the same locale everywhere. In some rare cases it may format the number in a different way which may cause problems.

See the `locales` param [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString) for details.